src/main/java/com/startstepszalando/ecommerceshop/user/service/UserService.java

```diff
@@ -0,0 +1,27 @@
+package com.startstepszalando.ecommerceshop.user.service;
+
+import com.startstepszalando.ecommerceshop.user.dto.UserRegistrationDto;
+import com.startstepszalando.ecommerceshop.user.model.User;
+import com.startstepszalando.ecommerceshop.user.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+    public void registerUser(UserRegistrationDto registrationDto) {
+        User user = new User();
+        user.setName(registrationDto.getName());
```

Comment:
Since you used @Builder for User model, u can use builder over setting. 
User.builder()
        .name(xyz).. and so on